pages.ne/common/cp.md

```diff
@@ -0,0 +1,28 @@
+# cp
+
+> फाइलहरू र डिरेक्टोरीहरू सार्नुहोस।
+> थप जानकारी: <https://www.gnu.org/software/coreutils/cp>.
+
+- एउटा स्थान बाट अर्को स्थानमा फाइलहरु सार्नुहोस :
```

Comment:
Is this space required in Nepali?
```suggestion
- एउटा स्थान बाट अर्को स्थानमा फाइलहरु सार्नुहोस:
```

Reply:
I don't think it is required. We generally don't include space in line endings in Indic languages.